src/api/handlers/gateway.ts

```diff
@@ -1,5 +1,5 @@
 import { RequestManager } from "../../rest/request_manager.ts";
-import { Application, DiscordBotGatewayData } from "../../types/mod.ts";
+import { DiscordBotGatewayData } from "../../types/mod.ts";
```

Comment:
The name of the file has been changed, this is an irrelevant change as well.

Reply:
You mean we should keep `misc.ts`?